File: Data Structures/Arrays/Cpp/Calculate_the_avg.cpp
// Write a program to read an array and calculate the average of the elements in the array.

#include<iostream>
using namespace std;

int main(){
    int Array[5];
    for(int i = 0; i < 5; i++){
        cin >> Array[i];
    }
    int sum = 0;
    for(int i = 0; i < 5; i++){
        sum += Array[i];
    }
    double average = (double)sum / 5;
    cout << "Average of the elements present in the array: " << average << endl;

    return 0;
}

File: Data Structures/Arrays/Java/Calculate_the_avg.java
// Write a program to read an array and calculate the average of the elements in the array.

import java.util.Scanner;

public class Calculate_the_avg{
    public static void main(String[] args){
        Scanner read = new Scanner(System.in);
        int[] array = new int[10];
        for(int i = 0; i < 10; i++){
            array[i] = read.nextInt();
        }
        int sum = 0;
        for(int i = 0; i < 10; i++){
            sum += array[i];
        }
        double avg = sum/10;
        System.out.println("The average of the elements of the array is: "+ avg);
        read.close();

    }
}